src/Input.jsx

```diff
@@ -0,0 +1,15 @@
+export default function Input({
+  nameValue, categoryValue, placeValue,
+  onChange, onClick,
+}) {
+  return (
+    <div>
+      <input onChange={onChange} value={nameValue} type="text" placeholder="이름" />
```

Comment:
```suggestion
      <input 
        onChange={onChange}
        value={nameValue}
        type="text"
        placeholder="이름"
      />
```

비교했을 때 어떠신것 같나요?
저는 속성이 2개 이상일 때는 줄바꿈해주면 코드가독성이 더 좋다고 생각하고 코드를 정리해주는 편이에요.
코드 가독성을 고려한 자기만의 코드 정리 기준을 세우시면 좋아요.

Reply:
코드 가독성을 고려해서 코드 정리도 해보도록 할게요! 코멘트 감사드립니다. 👏